src/main/java/com/flixmovies/imdb/dto/SingleMovieDto.java

```diff
@@ -0,0 +1,20 @@
+package com.flixmovies.imdb.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class SingleMovieDto {
```

Comment:
This is the same case as for the class AllMoviesDto. Try to name it thinking of the composition of the class: MovieDetailsDto, FullMovieDetailsDto, MovieDto, CompleteMovieDto...